torch/_dynamo/variables/lists.py

```diff
@@ -52,6 +64,9 @@ def cls_for(obj):
             torch.Size: SizeVariable,
             tuple: TupleVariable,
             set: SetVariable,
+            odict_values: ListVariable,
+            torch.nn.ParameterList: ListVariable,
+            torch.nn.ModuleList: ListVariable,
```

Comment:
No these all look sus. I would not expect ListVariable to accurately model any of these

Reply:
This is preserving behavior in builder. Its pure code motion